edx_rest_api_client/client.py

```diff
@@ -91,26 +91,35 @@ class OAuthAPIClient(requests.Session):
     authentication method, given a client id and client secret. The underlying implementation
     is subject to change.
     """
-    oauth_uri = '/oauth2/access_token'
-
     def __init__(self, base_url, client_id, client_secret, **kwargs):
         """
         Args:
-            base_url (str): base url of LMS instance
+            base_url (str): base url of LMS instance or
+                `BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL` which already includes the path `/oauth2`.
```

Comment:
Maybe examples to clarify what you mean by this?

Reply:
I’ll add that next year. 😉

Reply:
Done.

Reply:
Done.